crates/astria-conductor/src/celestia/builder.rs

```diff
@@ -319,14 +308,226 @@ fn wss_scheme() -> Scheme {
     "wss".parse::<_>().unwrap()
 }
 
+#[derive(Debug, PartialEq, Eq)]
+struct Endpoints {
+    http: Uri,
+    websocket: Uri,
+}
+
+impl FromStr for Endpoints {
+    type Err = eyre::Report;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        // The docs promise that inputs like `<authority>/<scheme>` work,
+        // even though that is not a valid URI because it's lacking a scheme.
+        // If parsing `s` as a URI fails the code below prepends the input with
+        // `dummy://{s}` and tries parsing that instead. If that also fails, the
+        // error of the first parse is returned.
+        //
+        // XXX: This is a hack to not have a breaking change in the example config.
```

Comment:
Flagging this. We should probably just not do this and instead just have 2 env vars, one for HTTP, one for WS.

Reply:
yeah that would be better